homeassistant/components/soma/sensor.py

```diff
@@ -0,0 +1,40 @@
+"""Support for Soma sensors."""
```

Comment:
This file is not covered by tests, either provide tests or add this file to the coverage ignore list.